frontend/src/components/Hacknight.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-container fluid text-xs-center>
+    <v-row align="center" justify="center">
```

Comment:
To avoid in-line styling instead you could use:

`<v-row class="X`></row>`

And in style section

`<style lang="scss" scoped>`
`.X {`
`  display: flex;`
`  align-content: center;`
`  justify-content: center;`
`}`
`</style>`

Reply:
It is not inline styling - these are props provided by Vuetify.